### ISSUE TITLE
Question about getting blocked

### ISSUE THREAD

**Issue by Gykonik** [Der-Henning/tgtg#268]
Hey Henning, thanks for your great work on this project! Do you have experience with TGTG blocking you after some time of scraping? I'm desperate because I'm always late for some stores, but would also hate to have my account banned.

**Comment by Der-Henning** [comment]
Hi @Gykonik. I assume you didn't use any TGTG scrapers until now? I didn't hear of anybody receiving a permanent ban on their TGTG account. TGTG blocks requests for a limited time if you make too many requests. In this case, it might be necessary to solve a captcha (this find all traffic lights thingy) on your phone so you can use the TGTG app again. 

In the worst case, when you receive a permanent ban, you could create a new account.

In conclusion, I cannot make any binding statements, as you violate TGTGs terms and conditions of their service if you use any API scrapers. You simply shouldn't overdo it by, for example, creating a paid service for others making thousands of requests :)

**Comment by Der-Henning** [comment]
Closing due to inactivity. If you have any further questions or problems, feel free to reopen or create a new issue.
